### PR TITLE
Use same LogSymbol.String for all platforms

### PR DESCRIPTION
**Pull request by plutov**
I just found that these symbols work well in Unix also, so I removed unnecessary `if`.

### CODE REVIEW

File: symbols.go [hunk 1]
```diff
@@ -14,28 +12,16 @@ const (
 )
 
 func (s LogSymbol) String() string {
-	if runtime.GOOS == "windows" {
```
Comment (gernest) [inline symbols.go]: This was supposed to be `runtime.GOOS != "windows"` because the symbols don't work on windows.

File: symbols.go [hunk 2]
```diff
@@ -14,28 +12,16 @@ const (
...
-		case INFO:
-			return "i"
-		case SUCCESS:
-			return "v"
-		case WARNING:
-			return "!!"
-		case ERROR:
-			return "x"
-		}
+	switch s {
```
Comment (gernest) [inline symbols.go]: just remove this , only one fix is needed  see my previous comment.

File: symbols.go (hunk 1 above)
Comment (plutov) [reply symbols.go]: Ah, got it :)

**Comment by plutov** [comment]
Done, please check.

**Comment by gernest** [comment]
Thanks

**Pull request merged**